test/gatekeeper/test-mutations.yaml

```
@@ -0,0 +1,35 @@
+---
+apiVersion: mutations.gatekeeper.sh/v1beta1
+kind: Assign
+metadata:
+  name: allow-privilege-escalation
+spec:
+    match:
+      scope: Namespaced 
+      kinds:
+        - apiGroups: ["*"]
+          kinds: ["Pod"]
+    applyTo:
+    - groups: [""]
+      kinds: ["Pod"]
+      versions: ["v1"]
+    location: "spec.containers[name:*].securityContext.allowPrivilegeEscalation"
+    parameters:
+      pathTests:  
+      - subPath: "spec.containers[name:*].securityContext.allowPrivilegeEscalation"
+        condition: MustNotExist
+      assign:
+        value: false
+
+---
+apiVersion: mutations.gatekeeper.sh/v1alpha1
+kind: AssignMetadata
+metadata:
+  name: label-location
+spec:
+  match:
+    scope: Namespaced
+  location: "metadata.labels.location"
+  parameters:
+    assign:
+      value: "Florida"
```

```suggestion
      value: "Florida"
 
```

Add a newline at the end of the file